src/region/RegionList.tsx

```diff
@@ -20,11 +22,18 @@ export function RegionList({ activeId, annotations, className, onSelect, scale }
         return shapeA.height * shapeA.width > shapeB.height * shapeB.width ? -1 : 1;
     });
 
+    const refs = sortedAnnotations.map(() => React.createRef<HTMLAnchorElement>());
```

Comment:
Do we need the refs for all the annotation targets or can we use the ref of the `svg` element?

Reply:
It's a good question, I'll see if I can make do with just the `svg`